src/list.rs

```diff
@@ -14,6 +14,14 @@ use core::ops;
 use slab::Slab;
 
 impl crate::Inner {
+    /// Locks the list.
+    fn lock(&self) -> Option<ListGuard<'_>> {
```

Comment:
this method ought to be named `try_lock`, right?

Reply:
I renamed it to `try_lock` in a later commit